### PR TITLE
Simplified Language-Selection

### PR DESCRIPTION
**Pull request by Zukye**
Languages are now provided in a list loaded from available property files instead of comparing strings when filling the language selection dropdown. 
Initially shown language is now correct.

### CODE REVIEW

File: src/main/java/org/openbase/bco/bcozy/view/ObserverButton.java [hunk 1]
```diff
@@ -1,17 +1,17 @@
 /**
  * ==================================================================
- *
+ * <p>
```
Comment (DivineThreepwood) [inline src/main/java/org/openbase/bco/bcozy/view/ObserverButton.java]: please remove this artifacts

File: src/main/java/org/openbase/bco/bcozy/util/Languages.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+    /**
+     * Initializes Languages, either from available property-files or uses ["en_US","de_DE"].
+     */
+    private Languages() {
+        List<Locale> bundles = getLocalesFromBundles();
+        if (!bundles.isEmpty()) {
+            this.languages = bundles.stream().map(this::fromLocale).collect(Collectors.toList());
+        } else {
+            this.languages = Arrays.asList(english, fromLocale(Locale.GERMANY));
```
Comment (DivineThreepwood) [inline src/main/java/org/openbase/bco/bcozy/util/Languages.java]: Why is the fallback language forced to german?
Can this be replaced by `Locale.getDefault(`) ?

File: src/main/java/org/openbase/bco/bcozy/view/ObserverText.java [hunk 3]
```diff
@@ -48,32 +60,40 @@
...
-    @Override
-    public void update(final Observable observable, final Object arg) {
-        setIdentifier(identifier);
+    public SimpleStringProperty identifierProperty() {
+        return identifier;
+    }
+
+    public void setApplyOnNewText(Function<String, String> applyOnNewText) {
+        this.applyOnNewText = applyOnNewText != null ? applyOnNewText : Function.identity();
+        this.update(null, null);
```
Comment (DivineThreepwood) [inline src/main/java/org/openbase/bco/bcozy/view/ObserverText.java]: null should not be used as default argument.
If possible introduce a additional methods without arguments.

**Review by DivineThreepwood — CHANGES_REQUESTED** [review]
Nice enhancement!
Only some minor code issues.

**Comment by Zukye** [comment]
all issues fixed

**Review by DivineThreepwood — APPROVED** [review]

**Pull request merged**